script.js

```diff
@@ -362,3 +363,22 @@ function screenMessage(message){
   noLoop();
 }
 
+function findCurrWord(){
+  if(word1Guessed == false){
```

Comment:
You should pretty much never compare directly to boolean literals. Since `word1Guessed` is already a boolean just use it, e.g.

```
if (!word1Guessed) { ...
```

This makes the code read more like (slightly out of order) English: "if not word1Gussed". It also avoids the bug where you accidentially type `word1Guessed = false` (note the single `=` rather than `==`) creating an expression which is always false, since the value of an assignment is the value being assigned.

Reply:
Probably a more important change to make here, however, is whenever you have a set of variables like this with numbers in their names, that's crying out to be a single array of values.

Reply:
Ah ok, I was wondering how to make that less annoying. Would an array that just goes in order [true, false, false, etc.] for whether they've been guessed work better?

Reply:
Actually, for this I don't think you even need an array. Why not just keep a count of what word you're on. Start at 0 and increment it each time they make a guess. And then use that variable as an index into the right row in your div or into an array of words or however you're storing the guesses.